docs/getting-started/interfaces.md

```diff
@@ -30,7 +30,7 @@ $ conda activate shapeworks
 ShapeWorksStudio is a cross-platform graphical user interface (GUI) to support the standard shape analysis workflow needed by most ShapeWorks users, enabling a wide variety of research scenarios. It is currently supporting image-based grooming and is actively under development to support surface meshes, more sophisticated grooming operations (including user annotations), and offline processing.  ShapeWorksStudio enables real-time parameter tuning and visualization of the optimization process and statistical analyses.
 
 
-[![ShapeWorksStudio](../img/getting-started/studio_thumbnail.png)](https://youtu.be/RX4uSB_ot3U "ShapeWorksStudio - Click to Watch!")
+<p><video src="https://sci.utah.edu/~shapeworks/doc-resources/mp4s/studio_demo.mp4" autoplay muted loop style="width:100%"></p>
```

Comment:
These are great! No more YouTube ads when trying to rtfm. :P 
Where are the videos stored at SCI? Is this location noted anyway in the documentation?

Reply:
Good point. That was on the [internal todo list](https://sciinstitute.github.io/ShapeWorks/todo/) for docs. I submitted an issue now #703.